### PR TITLE
Replace GOGS with GITEA in variable names

### LINKED ISSUE #87: Replace all GOGS_* variables
The title already says everything, there are still variables with this making schema.

### PR DESCRIPTION
**Pull request by strk**
Still use GOGS_WORK_DIR and GOGS_CUSTOM env variables
as a fallback if the equivalent GITEA_* are not set,
warning user about the need for change.

Also change "gogs" to "gitea" in webhook type name

Closes #87

### CODE REVIEW

**Comment by xinity** [comment]
i'll probably rewrite the Dockerfile.
still LGTM to me

**Comment by strk** [comment]
The WebHook name is still "gogs" and route includes `/gogs/` component, so the API and templates are unchanged

File: models/pull.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ var PullRequestQueue = sync.NewUniqueQueue(setting.Repository.PullRequestQueueLe
 type PullRequestType int
 
 const (
-	PULL_REQUEST_GOGS PullRequestType = iota
+	PULL_REQUEST_GITEA PullRequestType = iota
 	PLLL_ERQUEST_GIT
```
Comment (bkcsoft) [inline models/pull.go]: Cleaning this up as well would be nice :)

**Comment by bkcsoft** [comment]
Also maybe renaming the init-files? :)

**Comment by strk** [comment]
Typo cleaned

**Comment by strk** [comment]
Init files renamed

**Comment by bkcsoft** [comment]
What about this one? :smile: https://github.com/go-gitea/gitea/blob/master/scripts/systemd/gogs.service

**Comment by strk** [comment]
Also fixes #102 with latest commit (uses "gitea" in all init scripts)

File: models/pull.go [hunk 2]
```diff
@@ -26,8 +26,8 @@ var PullRequestQueue = sync.NewUniqueQueue(setting.Repository.PullRequestQueueLe
 type PullRequestType int
 
 const (
-	PULL_REQUEST_GOGS PullRequestType = iota
-	PLLL_ERQUEST_GIT
+	PULL_REQUEST_GITEA PullRequestType = iota
```
Comment (metalmatze) [inline models/pull.go:29]: Constants should be named lowerCamelCase for unexported ones and UpperCamelCase for exported ones.
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Comment (strk) [reply models/pull.go:29]: Can we use a separate PR for cleaning that up ? I'm just focusing on ^GOGS^GITEA here...

(and maybe we could use some automatici linting check to spot those)

**Comment by bkcsoft** [comment]
LGTM

File: models/pull.go (hunk 2 above)
Comment (strk) [reply models/pull.go:29]: Please see #110

**Pull request merged**